smac_planner/include/smac_planner/node_2d.hpp

```diff
@@ -82,7 +83,7 @@ class Node2D
    * @param cost_in The costmap cost at this node
    * @param index The index of this node for self-reference
```

Comment:
Has been removed

Reply:
thanks, there's also a ticket to go back and add docs to everything, I know over the last month or so I've been sloppy with it and needs a bunch of updates